packages/react-dom/src/client/ReactDOMHostConfig.js

```diff
@@ -247,32 +242,40 @@ export function getPublicInstance(instance: Instance): * {
   return instance;
 }
 
-export function prepareForCommit(containerInfo: Container): void {
+export function prepareForCommit(containerInfo: Container): Object | void {
   eventsEnabled = ReactBrowserEventEmitterIsEnabled();
   selectionInformation = getSelectionInformation();
+  let activeInstance;
   if (enableDeprecatedFlareAPI || enableUseEventAPI) {
     const focusedElem = selectionInformation.focusedElem;
     if (focusedElem !== null) {
-      const instance = getClosestInstanceFromNode(focusedElem);
-      if (instance !== null && isFiberInsideHiddenOrRemovedTree(instance)) {
-        dispatchBeforeDetachedBlur(focusedElem);
-      }
+      activeInstance = getClosestInstanceFromNode(focusedElem);
```

Comment:
This may give you null. So the return type seems a bit inconsistent (void or null or Fiber). Can we make it always `null | Fiber`?

Reply:
I originally did that, but then I had to change every renderer where we use this method to return null and it seemed a bit overdrawn. Happy to do that again though.

Reply:
Alternatively we could make it explicit in the type. Like `null | void | Object`. It's just easy to miss otherwise.

Reply:
We could also make it always return `undefined` instead of `null`.

Reply:
I've made it `null | Fiber`.